build/scripts/coverage.sh

```diff
@@ -5,8 +5,8 @@
 #
 # Code coverage generation
 CVG_EXCLUDE="${CVG_EXCLUDE:-test}"
```

Comment:
I think we can get rid of the CVG_EXCLUDE variable.  It is not used and don't think it works